rapids/test_rapids.py

```diff
@@ -0,0 +1,58 @@
+import random
+import unittest
+import os
+
+from parameterized import parameterized
+
+from integration_tests.dataproc_test_case import DataprocTestCase
+
+
+class RapidsTestCase(DataprocTestCase):
+    COMPONENT = 'rapids'
+    #INIT_ACTION = 'gs://dataproc-initialization-actions/rapids/rapids.sh'
+    #METADATA = 'INIT_ACTIONS_REPO=https://github.com/GoogleCloudPlatform/dataproc-initialization-actions.git,INIT_ACTIONS_BRANCH=master'
```

Comment:
Remove these 3 lines to make this ready for submission.

Reply:
done